db/schema.rb

```diff
@@ -660,6 +660,22 @@
     t.index ["user_id"], name: "index_steps_on_user_id"
   end
 
+  create_table "system_notifications", force: :cascade do |t|
+    t.string "title"
+    t.text "description"
+    t.string "modal_title"
+    t.text "modal_body"
+    t.boolean "show_on_login", default: false
+    t.datetime "source_created_at"
+    t.bigint "source_id"
+    t.datetime "last_time_changed_at"
```

Comment:
Should this field be `null: false`?

Reply:
I was thinking about this...
Do we see any benefits if the column is nullable? 
I thought to make it nullable for old migrated notifications, but not make sense or any advantage... 

Yeap, lets go with `null: false`...